.github/workflows/pr-lint.yml

```diff
@@ -19,7 +19,7 @@ jobs:
           submodules: recursive
       - uses: ./.github/actions/pr-lint-action
         with:
-          title-regex: "^(build|chore|ci|docs|feat|fix|perf|refactor|style|test|other)((.+))?:\\s.+"
+          title-regex: "^(build|chore|ci|docs|feat|fix|perf|refactor|style|test|other)(\\(.+\\))?(\\!)?:\\s.+"
```

Comment:
@nytai I escaped the parens here.. was that intentional originally? Also, do you know the reason for the last `s`?

Reply:
I think I coped this from superset-ui

Reply:
I think last s is white space?

Reply:
TBH, I'm also a little confused by this regex. I lifted it from this file https://github.com/apache/superset/blob/dc893fe1b3b86e2921bb77e08a531db46b3f8e5d/.github/prlint.json
that @mistercrunch added

Reply:
Ok, this is making more sense now. Looks like there's some yaml escaping in here too. Looks like there was a mistake in the original parens, so the previous regex allowed anything after the prefix and before the `:`, so `buildskjdsfsdfsdbjfds: dsfds` would have been valid 🤣. The last `\s` is the ensure there's white space after the `:`

Reply:
Ok, gotcha. yeah, the yaml escaping threw me off on the `\\s` at first too. Luckily the linter caught the new ones.